runners/portability/java/src/main/java/org/apache/beam/runners/portability/ExternalWorkerService.java

```diff
@@ -38,6 +44,7 @@
 public class ExternalWorkerService extends BeamFnExternalWorkerPoolImplBase implements FnService {
 
   private static final Logger LOG = LoggerFactory.getLogger(ExternalWorkerService.class);
+  private static final String PIPELINE_OPTIONS = "PIPELINE_OPTIONS";
```

Comment:
Nit: we can rename this variable to make its usage more obvious.
```suggestion
  private static final String PIPELINE_OPTIONS_ENV_VAR = "PIPELINE_OPTIONS";
```